_gcode/G0.md

```diff
@@ -62,14 +62,14 @@ parameters:
 examples:
   -
     pre:
-      The most basic example is to set a feedrate and them move the tool on the XY position.
+      The most basic move sets a feedrate and moves the tool to the given position.
     code:
       - G0 X12   ; move to 12mm on the X axis
       - G0 F1500 ; set the feedrate to 1500mm/minute
       - G1 X90.6 Y13.8 ; move to 90.6mm on the X axis and 13.8mm on the Y axis
   -
```

Comment:
Yes. We will need to do this throughout the documentation, wherever units are specified. After all, `steps_per_mm` is actually `steps_per_degree` on a SCARA machine, and on a delta it refers to the tower carriage speeds and not the cartesian XY speeds. So, we need to make note about all these sorts of things in a general way, such as "See the section 'Units and Measures'" for all codes that are affected by these global options.